documentation/release_notes.rst

```diff
@@ -8,6 +8,60 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.4
+=============
+
+New Features
+------------
+-  Added the range-based versions of the following algorithms: ``any_of``, ``adjacent_find``,
+   ``copy_if``,  ``remove_copy_if``, ``remove_copy``, ``none_of``, ``swap_ranges``, ``unique``, ``unique_copy``,
+   ``reverse``, ``reverse_copy``, ``rotate_copy``, ``replace_copy``,  ``replace_copy_if``.
```

Comment:
Would it be better if we had an alphabetical order here?

Reply:
Done.